src/test/java/org/opensearch/security/dlic/dlsfls/CCReplicationTest.java

```diff
@@ -232,7 +216,8 @@ public void testReplication() throws Exception {
         }
 
         try (Node node = new PluginAwareNode(false, tcSettings, Netty4Plugin.class, OpenSearchSecurityPlugin.class, MockReplicationPlugin.class).start()) {
-            waitOrThrow(node.client(), "hr-masking");
+            waitForInit(node.client());
+            node.client().execute(MockReplicationAction.INSTANCE, new MockReplicationRequest("hr-masking")).actionGet();
```

Comment:
These calls can be extracted into in a separate method:
```
void mockReplicationRequest(Client client, String replicationFor) {
    waitForInit(node.client());
    node.client().execute(MockReplicationAction.INSTANCE, new MockReplicationRequest(replicationFor)).actionGet();
}
```

Reply:
I brought back `waitOrThrow` and opted to change the implementation of `waitOrThrow` to use `waitForInit` to cut down our duplication.